src/storage/GraphStorageServiceHandler.cpp

```diff
@@ -89,14 +89,14 @@ GraphStorageServiceHandler::future_updateVertex(const cpp2::UpdateVertexRequest&
 // Edge section
 folly::Future<cpp2::ExecResponse>
 GraphStorageServiceHandler::future_addEdges(const cpp2::AddEdgesRequest& req) {
-    auto* processor = AddEdgesProcessor::instance(env_);
+    auto* processor = AddEdgesProcessor::instance(env_, &kAddEdgesCounters);
```

Comment:
No need to increase here, because the default value is used

Reply:
Just keep it, explicit is better. It seems all processor has default value.